dts/imx8mm-cgtsx8m-ultimain5.0-lvds-1024x600.dts

```diff
@@ -92,7 +91,30 @@
     };
 };
 
+&i2c2 {
+    /* Carrier data EEPROM (U12) */
+    carrierdata@57 {
+        compatible = "atmel,24c32";
+        reg = <0x57>;
+        pinctrl-names = "default";
+        pinctrl-0 = <&pinctrl_gpio4>;
+        pagesize = <32>;
+        wp-gpios = <&gpio4 10 0>;
+    };
+};
+
+/* Ultimainboard J8 debug console. We cannot use DMA, it uses proprietary firmware drivers.  */
+&uart4 { /* SER1 */
+    /delete-property/ dmas;
+    /delete-property/ dma-names;
+};
+
 / {
+    chosen { /* Investigate how this works, because this is usefull */
```

Comment:
this block can be removed

Reply:
Which part do you want removed? Just the comment, the 'chosen' block or the whole '/' block?

Reply:
The whole chosen block.